### PR TITLE
Added Galaxy A31 Device & Ui Overlays

### PR DESCRIPTION
**Pull request by DevZillion**

### CODE REVIEW

File: Samsung/A31/res/xml/power_profile.xml [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+        <value>400000</value>
+    </array>
+    <array name="cpu.active.cluster0">
+        <value>0.1</value>
+    </array>
+    <item name="cpu.idle">0.1</item>
+    <array name="memory.bandwidths">
+        <value>22.7</value>
+    </array>
+    <item name="battery.capacity">1000</item>
```
Comment (phhusson) [inline Samsung/A31/res/xml/power_profile.xml]: Doubt.

File: Samsung/A31/res/xml/power_profile.xml [hunk 2]
```diff
@@ -0,0 +1,67 @@
+<?xml version="1.0" encoding="utf-8"?>
+<device name="Android">
+    <item name="ambient.on">0.1</item>
+    <item name="screen.on">0.1</item>
```
Comment (phhusson) [inline Samsung/A31/res/xml/power_profile.xml]: A screen that consumes 0.1mA at max  brightness? awesome ! Sounds physically impossible but well...

File: Samsung/A31/res/xml/power_profile.xml [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+    <item name="camera.flashlight">0.1</item>
+    <item name="camera.avg">0.1</item>
+    <item name="gps.on">0.1</item>
+    <item name="radio.active">0.1</item>
+    <item name="radio.scanning">0.1</item>
+    <array name="radio.on">
+        <value>0.2</value>
+        <value>0.1</value>
+    </array>
+    <array name="cpu.active">
```
Comment (phhusson) [inline Samsung/A31/res/xml/power_profile.xml]: Sounds unlikely.

**Review by phhusson — CHANGES_REQUESTED** [review]
Is it tested? Your power_profile is utterly broken.

**Comment by DevZillion** [comment]
Sorry about the power_profile.xml.
An XDA gyide recommended to extract it from framework-res.apk, thx anyways for the requested changes and I'll fix them as fast as I can.

**Comment by DevZillion** [comment]
And yeah I tested it, what I don't know if what the power_profile overlay does so that's the most untested.

**Comment by DevZillion** [comment]
And yeah I tested it, what I don't know if what the power_profile overlay does so that's the most untested.

**Comment by DevZillion** [comment]
Finally Fixed

**Pull request merged**